controllers/broker_controller.go

```diff
@@ -79,18 +82,24 @@ func (r *BrokerReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 	// Get broker endpoint
 	err = brokerpackage.GetEndpoint(ctx, baseResource, r.Clientset, r.KubeConfig.Host)
 	if err != nil {
-		return ctrl.Result{}, ErrGetEndpoint(err)
+		err = ErrGetEndpoint(err)
+		r.Log.Error(err, "unable to get the broker endpoint")
+		return ctrl.Result{}, err
 	}
 
 	// Patch the broker resource
 	patch, err := utils.Marshal(baseResource)
 	if err != nil {
-		return ctrl.Result{}, ErrUpdateResource(err)
+		err = ErrUpdateResource(err)
+		r.Log.Error(err, "unable to update broker resource")
```

Comment:
Meaning that the resource couldn't be found or that it is present but the operator doesn't have permission or something else?